Web/Controller/LandingPages.hs

```diff
@@ -38,6 +38,34 @@ instance Controller LandingPagesController where
                     render EditView { .. }
                 Right landingPage -> do
                     landingPage <- landingPage |> updateRecord
+
+                    -- After we update the Landing page, we can set the order of the paragraphs.
+                    let params = paramListOrNothing @UUID "paragraphId"
+
+                    case catMaybes params of
+                        [] -> do
+                            -- No paragraphs to update.
+                            pure ()
+
+                        uuids -> do
+                            -- We need to update the weight of the paragraphs,
+                            -- So load them.
+                            landingPage <- fetchLandingPageWithParagraphs landingPageId
+
+                            -- Iterate over all paragraphs, and update the weight.
+                            -- @todo: Iterate over all paragraphs.
+                            forEach landingPage.paragraphCta \paragraph -> do
+                                let uuid = unpackId paragraph.id
+                                let weight = elemIndex uuid uuids |> fromMaybe 0
```

Comment:
![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/8242a0cb-235a-4652-8cba-8e5d905dfe9f)

But even if I remove this line, I'll get another error on the `updateRecord` below

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/e0f6e2d9-8b63-40fc-b9df-cb12efb68185)

Reply:
Typo in the variable name :/